### ISSUE TITLE
The page number overlay does not appear on the last page. (Inside the modal)

### ISSUE THREAD

**Issue by malgirlim** [motla/vue-document-editor#26]
** It is a sentence written using a translation program. Please forgive me.

**Describe the bug**
The page number overlay does not appear on the last page. (Inside the modal)

**To Reproduce**
1. Import components(Demo.vue) from within the Modal.
2. On the browser screen, you can see that the last page overlay appears.
3. Press the Print button.
4. You can see that there is no page number overlay on the last page.

**Expected behavior**
As a result of the test, the problem occurred only inside the Modal and no problem outside the Modal.
But I thought it might be a problem because it was different from the template environment I was using.

**Screenshots**
[If applicable, add screenshots to help explain your problem.
https://imgur.com/a/b1Ye5b2

**Desktop (please complete the following information):**
 - OS: Windows 11 Pro
 - Browser : Chrome, Edge
 - Version : Chrome : 111.0.5563.110 / Edge : 111.0.1661.44
 - Vue.js version : 3.2.45

**Additional context**
I am a novice developer, and I am working on a project using a template.
Templates use Vue3, TypeScript, and Vite.
This environment seems to be causing problems, but I couldn't find them with my ability.
Here's the github repertoire that reproduced my problem.
https://github.com/malgirlim/Print-Test

It's a really great solution, so I want to solve the problem and use it.
Please give me a little help. Thank you.

**Comment by motla** [comment]
Hi @malgirlim 

Thanks for your bug reports.
I'm sorry for the delay but I have been very busy lately and your bugs were not trivial to solve.

I couldn't find a way to solve the first one (missing page number). I think it has to do with the CSS formatting of your page... But what is weird is, when I call the DocumentEditor.vue `before_print()` function manually, and then I call `window.print()`, it works. But when I just call `window.print()` (which calls `before_print()` under the hood), it doesn't work.

However I managed to solve the blank page issue, it had to do with page overlays. I just released v2.3.0 (Vue3) and v1.5.0 (Vue2).

I let the issue open in case somebody finds a way to solve the first one...
Let me know if you manage to find a workaround.
